test/e2e/framework/util.go

```diff
@@ -1528,13 +1533,19 @@ func podsRunning(c *client.Client, pods *api.PodList) []error {
 	// are running so non-running pods cause a timeout for this test.
 	By("ensuring each pod is running")
 	e := []error{}
+	error_chan := make(chan error)
+
 	for _, pod := range pods.Items {
-		// TODO: make waiting parallel.
-		err := WaitForPodRunningInNamespace(c, pod.Name, pod.Namespace)
+		go WaitForPodRunningInNamespaceAsync(c, pod.Name, pod.Namespace, error_chan)
```

Comment:
do we expect `WaitForPodRunningInNamespaceAsync` to be used elsewhere? otherwise I'm slightly in favor of not exporting another util function. maybe just
```go
go func() {
  error_chan <- WaitForPodRunningInNamespace(...)
}
```